### ISSUE TITLE
Make properties for the font of the title available

### ISSUE THREAD

**Issue by team-moeller** [team-moeller/better-access-charts#99]
The documentation mentions the following font related properties:

- fontsize
- fontcolor
- fontfamily
- fontstyle

**Comment by team-moeller** [comment]
Necessary code for the html string :

```
title: {
        fontFamily: 'Algerian',
	fontSize: 16 ,
	fontColor: 'rgb(255, 99, 132)',
	fontStyle: 'italic bold'
},
```

**Comment by team-moeller** [comment]
Released with version 2.21.05
